gradle/verification-metadata.xml

```diff
@@ -84,11 +84,21 @@
             <sha256 value="06d4035a0b2373f0fef529de0e6f81ff43ef7a36ecf476e5d5f1aecba8ee4fad" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.amazonaws" name="aws-java-sdk-core" version="1.12.270">
+         <artifact name="aws-java-sdk-core-1.12.270.jar">
+            <sha256 value="4e41d9f54606151674fc550e5e6291b0ddf917d55a2a3465a45a4e6ac98c9f8f" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="com.amazonaws" name="aws-java-sdk-ec2" version="1.11.749">
          <artifact name="aws-java-sdk-ec2-1.11.749.jar">
             <sha256 value="293cc44d21aac78f5d974992606e7c4bbff20e8d0b897d0f13476424c59df843" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.amazonaws" name="aws-java-sdk-ec2" version="1.12.270">
+         <artifact name="aws-java-sdk-ec2-1.12.270.jar">
+            <sha256 value="faadf443751822e205338e80d2cea5eabd6373c1c3cef6348c24809ca82a9dd0" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="com.amazonaws" name="aws-java-sdk-s3" version="1.11.749">
          <artifact name="aws-java-sdk-s3-1.11.749.jar">
```

Comment:
Aren't we suppose to remove these?

Reply:
Apparently, Gradle doesn't clean them up automatically https://docs.gradle.org/current/userguide/dependency_verification.html#sec:verification-metadata-hygiene

Reply:
I've manually removed the outdated checksums